pkg/mattermost/mattermost_v1beta.go

```diff
@@ -94,6 +100,26 @@ func configureMattermostService(service *corev1.Service) *corev1.Service {
 	return service
 }
 
+func configureMattermostServiceNodePort(service *corev1.Service) *corev1.Service {
+	service.Spec.Ports = []corev1.ServicePort{
```

Comment:
Instead of doing same thing as `configureMattermostService`, can we add a separate function to set port? 

Both `configureMattermostServiceNodePort` & `configureMattermostService` will call that function to set these ports.